rs/execution_environment/src/scheduler/round_schedule.rs

```diff
@@ -229,4 +231,77 @@ impl RoundSchedule {
 
         (canisters_partitioned_by_cores, canisters)
     }
+
+    /// Finishes canister execution on thread.
+    pub fn finish_canister_execution_on_thread(
```

Comment:
Nit: This is not a particularly useful doc comment. What exactly does "finishing canister execution" mean beyond "no longer executing the canister" (which would be an empty function)?

Also the "on thread" doesn't really add much information.